Civi/Api4/Service/Spec/Provider/IsCurrentFieldSpecProvider.php

```diff
@@ -71,4 +86,17 @@ public static function renderIsCurrentSql(array $field): string {
     return "IF($isActive = 1 AND ($startDate <= '$todayStart' OR $startDate IS NULL) AND ($endDate >= '$todayEnd' OR $endDate IS NULL), '1', '0')";
   }
 
+  /**
+   * Render the sql clause to filter on expires date.
+   *
+   * @param array $field
+   *
+   * return string
+   */
+  public static function renderNonExpiredSql(array $field): string {
+    $endDate = substr_replace($field['sql_name'], 'expires_date', -11, -1);
```

Comment:
@eileenmcnaughton is it me or does this field name feel odd, I feel like maybe expiry_date might make a little more sense i'm just not feeling it with expires for some reason

Reply:
@seamuslee001 yeah fair point. When I added `expires_date` to `UserJob` I chose that name because the field already existed in `Search` - I'm not on board for changing it now - but it is a good reason to bike shed up front